### ISSUE TITLE
Adding a folder path roms pre-built

### ISSUE THREAD

**Issue by MHBdev2** [Swordfish90/Lemuroid#856]
I would like to know if it is possible to embed in the code in advance the path of the folder from which the application loads games (that is, it will be the default for loading games)

**Comment by X64X2** [comment]
Yes if you're willing to do it yourself and submit a PR highlighting that .

**Comment by MHBdev2** [comment]
I am not experienced enough in Kotlin (more with JAVA) to do such a thing alone,
I was going to ask if someone familiar with the project could direct me where to start
